hazelcast/src/main/java/com/hazelcast/jet/impl/connector/ReadJdbcP.java

```diff
@@ -104,7 +105,7 @@ public static <T> ProcessorMetaSupplier supplier(
 
         return ProcessorMetaSupplier.forceTotalParallelismOne(
                 SecuredFunctions.readJdbcProcessorFn(connectionURL,
-                        (ctx) -> new DataSourceFromJdbcUrl(connectionURL),
+                        (ctx) -> DataStoreHolder.closing(new DataSourceFromJdbcUrl(connectionURL)),
```

Comment:
One more place, wrapping seems unneeded, since `DataSourceFromJdbcUrl` is not Closable

Reply:
fixed https://github.com/hazelcast/hazelcast/pull/22372/commits/d7fc71874c0595930eba531be3c8ddc05b4623bd